go.mod

```diff
@@ -9,13 +9,14 @@ require (
 	github.com/btcsuite/btcd v0.21.0-beta
 	github.com/btcsuite/btcutil v1.0.2
 	github.com/confio/ics23/go v0.6.3
+	github.com/cosmos/cosmos-sdk v0.42.4
```

Comment:
There's no need to import `cosmos-sdk` here, we can use the `cosmos.tx.v1beta1.Tx` type from the `tendermint-protobuf-def` repository.

Reply:
oh you might be running into an issue here as the go_package in cosmos.tx.v1beta1 etc has 

`option go_package = "github.com/cosmos/cosmos-sdk/types/tx"` instead of just a figment one and its trying to pull it in from that. I'm fairly sure I had this issue with the protobuf Slimming.

Reply:
We don't build the go binding for cosmos type currently in tendermint-protobuf-def repo. I used the cosmos-sdk version 
 from gaia tendermint dependencies. The tx.proto file in tendermint-protobuf-def repo is from 0.45.1.

Reply:
Alright, let's see if importing types is a viable approach and keep it the way it is for now.

BTW, `cosmos-sdk` imports `tendermint`, is that an issue for us?